### PR TITLE
Marj

### PR DESCRIPTION
**Pull request by MarjorieBR**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Women In Tech</title>
     <link rel="stylesheet" href="./css/stylesheet.css">
-    <script src="./js/app.js"></script>
```
Comment (AmouletteB) [inline index.html]: This had been fixed

**Comment by AmouletteB** [comment]
Hi Marjorie,

Thanks for this. I think you've been using an old version of the file to
edit thought as changes have been made by Honor and me. You would need to
go on your github desktop, pull origin to download last version, and then
commit your changes so I can review. Alternatively, you can send me the
code for your testimonials page and thank you page and I'll add them in.

Cheers,

Amelie

**Review by AmouletteB — APPROVED** [review]

**Review by AmouletteB — APPROVED** [review]

**Comment by MarjorieBR** [comment]
Hi Amelie,

Thank you very much. As you can see, I'm very new at this. 

Cheers,

Marj

**Comment by AmouletteB** [comment]
No worries I'll get it fixed. If you can just send me the html / css files
in attachments that would be great. As Honor has done basically :)

**Comment by AmouletteB** [comment]
All fixed, thank you!

**Pull request merged**